xbmc/cores/VideoPlayer/VideoRenderers/windows/RendererDXVA.cpp

```diff
@@ -160,16 +158,17 @@ bool CRendererDXVA::Configure(const VideoPicture& picture, float fps, unsigned o
     // create processor
     m_processor = std::make_unique<DXVA::CProcessorHD>();
     if (m_processor->PreInit() && m_processor->Open(picture) &&
-        m_processor->IsFormatSupported(dxgi_format, support_type))
+        m_processor->IsFormatSupportedInput(dxgi_format))
     {
       m_intermediateTargetFormat = CalcIntermediateTargetFormat(picture, tryVSR);
       if (m_processor->SetOutputFormat(m_intermediateTargetFormat))
       {
         if (CServiceBroker::GetLogging().IsLogLevelLogged(LOGDEBUG) &&
             CServiceBroker::GetLogging().CanLogComponent(LOGVIDEO))
-          m_processor->ListSupportedConversions(dxgi_format, dest_format, picture);
+          m_processor->LogSupportedConversions(dxgi_format, m_intermediateTargetFormat, picture);
```

Comment:
Seems variable `dest_format` is not used anymore then can be removed (line 144):

`const DXGI_FORMAT dest_format = DX::Windowing()->GetBackBuffer().GetFormat();`

Reply:
true.